Source/Charts/Renderers/BarChartRenderer.swift

```diff
@@ -71,7 +71,7 @@ open class BarChartRenderer: BarLineScatterCandleBubbleRenderer
             }
         }
 
-        _buffers = zip(_buffers, barData.dataSets).map { buffer, set -> Buffer in
+        _buffers = zip(_buffers, barData).map { buffer, set -> Buffer in
```

Comment:
@jjatie what you think is the proper time we remove `dataSets` var? Like v4.1?

Reply:
I'd say move it to `obsoleted` in 4.1, then transition it out in 5?

Reply:
I think 5 can go a little faster that 4. I think a good goal for 5 would be to move all objc support into a compatibility library, and remove it from the core project. This would allow us to use Swift's generics system, protocols, and structs more effectively in the core lib, and expose more type safe API in the objc compat library too!